Src/PCompiler/CompilerCore/Parser/PLexer.g4

```diff
@@ -63,6 +63,9 @@ WHILE     : 'while' ;
 WITH      : 'with' ;
 CHOOSE    : 'choose' ;
 
+// denote enum as pred
```

Comment:
This got added upstream from recent changes by @lmpick relating to adding predicate disjunction tracking from [here](https://github.com/aman-goel/P/commit/41b794d8d58d34167210641aa418409c3fb9eb28#diff-ecd29283bed85bd7652cea7a4962fe2a3f3b053b5bc46634b75a35ccbbb74091), by adding the concept of a ``pred enum`` that is internally modeled in PSym using a String value summary (instead of an Integer value summary). I believe @lmpick might be able to provide a better context and whether to retain or not.

Reply:
This was added for predicate abstraction in the `fp` branch. I don't think it should be included here.

Reply:
@aman-goel did you pull the changes from `fp` branch into the master? I would not recommend that. I dont think the `fp` branch changes were tested enough. I would remove the changes from `fp` branch and create a new PR if its not too much work.